projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -95,13 +118,15 @@ export default function MediaSection() {
 		[]
 	);
 
+	const onDismissClick = useCallback( () => setShotNotice( false ), [] );
+
 	return (
 		<div className={ styles.wrapper }>
 			<MediaUploadCheck>
 				<MediaUpload
 					title={ ADD_MEDIA_LABEL }
 					onSelect={ onUpdateMedia }
-					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					allowedTypes={ mediaValidator.allowedImageTypes }
```

Comment:
I know this doesn't work currently anyway, but do you think we should allow all possible types, but then warn people with the notice? If types come and go as the images are uploaded, it could be confusing.

Reply:
The bug is that you can upload anything, but if you open up the library it actually hides everything else like .zip, which I think is helpful, because they only see images from their media lib. What do you think?